glean_parser/schemas/pings.1-0-0.schema.yaml

```diff
@@ -46,8 +46,79 @@ additionalProperties:
         When `true`, include the `client_id` value in the ping.
       type: boolean
 
+    notification_emails:
+      title: Notification emails
+      description: |
+        **Required.**
+
+        A list of email addresses to notify for important events with the
+        ping or when people with context or ownership for the ping need to
+        be contacted.
+      type: array
+      minItems: 1
+      items:
+        type: string
+        format: email
+
+    bugs:
+      title: Related bugs
+      description: |
+        **Required.**
+
+        A list of bugs (e.g. Bugzilla and Github) that are relevant to this
+        ping, e.g., tracking its original implementation or later changes to
+        it.
+
+        If a number, it is an ID to an issue in the default tracker (e.g.
+        Mozilla's Bugzilla instance). If a string, it must be a URI to a bug
+        page in a tracker.
+      type: array
+      minItems: 1
+      items:
+        anyOf:
+          - type: integer
+          - type: string
+            format: uri
+
+    data_reviews:
+      title: Review references
+      description: |
+        **Required.**
+
+        A list of URIs to any data collection reviews relevant to the ping.
+      type: array
+      items:
+        type: string
+        format: uri
+
+    disabled:
+      title: Disabled
+      description: |
+        If `true`, the ping is disabled, and attempting to send it is a no-op.
+      type: boolean
+      default: false
+
+    expires:
```

Comment:
I think @georgf had strong opinions on this one. Let's make sure to check with him before merging this.

Reply:
Yeah, i'd love to have a design conversations on this specific feature first.
I'm not sure how this should work together with probe expiry attributes without surprising users.
I also don't understand yet what a typical user story around a time limited ping is.

For me my model is currently more that pings are more transport - as such it matters what metrics in them expire or not, not that the ping expires.

Unless we're already clear on the above points (maybe i just missed that?), i would like us to avoid adding the complexity here and think about the impact first.